doc/api/console.md

```diff
@@ -392,6 +392,25 @@ are identified by a unique `label`. Use the same `label` when calling
 [`console.timeEnd()`][] to stop the timer and output the elapsed time in
 milliseconds to `stdout`. Timer durations are accurate to the sub-millisecond.
 
+### console.timeLog([label][, ...data])
```

Comment:
It seems this should go after `### console.timeEnd(label)` section ABC-wise.

Reply:
All `console.` methods seem to be in ABC order [according to the TOC](https://nodejs.org/download/nightly/v11.0.0-nightly201806134c166935ae/docs/api/console.html).

Reply:
Right. I didn't see that inspector methods were in another section